lib/portage/tests/emerge/conftest.py

```diff
@@ -406,7 +406,7 @@ def async_loop():
     yield asyncio._wrap_loop()
 
 
-@pytest.fixture(params=SUPPORTED_GENTOO_BINPKG_FORMATS, scope="module")
+@pytest.fixture(params=SUPPORTED_GENTOO_BINPKG_FORMATS, scope="function")
 def playground(request, tmp_path_factory):
```

Comment:
When I ran the lib/portage/tests/emerge tests on my laptop with this change, the time only went up from 3:59 to 4:08, and the number of instantiated playground fixtures went up from 6 to 24. Seems worthwhile to improve cleanup.

Reply:
Thanks, I didn't even check the run time.